unity-client/Assets/Scripts/MainScripts/DCL/Components/Video/DCLVideo.cs

```diff
@@ -0,0 +1,37 @@
+using DCL.Components;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.Video;
+
+namespace DCL.Components
+{
+    public class DCLVideo : BaseComponent
```

Comment:
Remember to add some tests to ensure the component is at least created correctly.